.gitignore

```diff
@@ -18,3 +18,12 @@ __pycache__/
 
 #sphinx build
 docs/_build/
+
+.venv
+.vscode
+issues
+results
+output
+test*
```

Comment:
This will prevent new `test_` files from being added to version control.

Also, why do we need to ignore `issues`, `results`, `output`, `gps_output`, and `plan*`?

Reply:
Sorry this was by accident